apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -896,20 +931,31 @@ export class PretixPipeline implements BasePipeline {
       const { emailClaim } =
         await this.credentialSubservice.verifyAndExpectZupassEmail(req.pcd);
 
-      if ((this.definition.options.semaphoreGroups ?? []).length > 0) {
```

Comment:
In `PipelineAPISubService.handleGetPipelineInfo`, we only fetch consumer data for display in the dashboard if Semaphore groups are configured. However, with this change, the existence of consumer data no longer depends on whether Semaphore groups are configured, so we could change this:

https://github.com/proofcarryingdata/zupass/blob/cb15a262de5da830706117fce6dab2bdf83d0032/apps/passport-server/src/services/generic-issuance/subservices/PipelineAPISubservice.ts#L143-L146

For consistency perhaps the Lemonade pipeline should also save consumer data independently of whether Semaphore groups are configured.

Reply:
implemented both suggestions